en/gps_compass/rtk_gps_fem_mini2.md

```diff
@@ -0,0 +1,61 @@
+# Femtones MINI2 receiver
+
+[MINI2 Receiver](http://www.femtomes.com) supports the absolute accuracy positioning of Global Navigation Satellite System (GNSS).
+This delivers the quickest and most reliable RTK initializations for centimeter level positioning.
+Mainly facing to applications of high-precision positioning, navigation and mapping, etc. 
+Customers benefit from the Ethernet connectivity available on the board, allowing high speed data transfer and configuration via standard web browsers.
+Serial port, USB and CAN are also supported.
+
+<img src="http://www.femtomes.com/image/PX4/MINI_II_Receiver.jpg" width="500px" title="mini2" />
+
+## Required Firmware Options
+
+The following firmware options need to be selected when buying the device:
+
+- 5Hz 10Hz 20Hz 
+- INS 
+- HEADING
+- OBS
+- RTK
+- BASE 
+
+## Purchase
+
+* [http://www.femtomes.com](http://www.femtomes.com)
+* [sales@femtomes.com](mailto:sales@femtomes.com)
+* TEL: +86-10-53779838
+
+## Functional Ports
+
+<img src="http://www.femtomes.com/image/PX4/MINI_II_1.jpg" width="500px" title="mini2port" />
+
+## Wiring and Connections
+

```

Comment:
Above you say "Customers benefit from the Ethernet connectivity available on the board, allowing high speed data transfer" but this only covers connecting via UART. Nor CAN.

1. Do you provide any cables to make it easy to connect? If so, what are they?
2. It is useful to create a little table like this one to make the wiring very easy for usrs: http://docs.px4.io/master/en/gps_compass/rtk_gps_cuav_c-rtk.html#pinout I would normally do it with Pixhawk 4 as the flight controller.
3. What have you tested this for with PX4? Ie has this been tested on CAN vs PX4 running on what hardware. Same with Ethernet. If you used Ethernet what hardware and how was it set up?

Reply:
The Ethernet connectivity is for PC I suppose.

Reply:
> The Ethernet connectivity is for PC I suppose.

The docs should state how this works and what has been tested on PX4, not "generically". So something like "For PX4 you can connect using serial and X, Y ,Z. Ethernet can be used to configure the RTK from a PC (via a web interface).

Does that make sense?